src/SdkCommon/ClientRuntime.Azure/ClientRuntime.Azure.Tests/CloudErrorJsonConverterTest.cs

```diff
@@ -26,6 +26,14 @@ public void TestCloudErrorDeserialization()
                             ""target"": ""$search"",
                             ""message"": ""$search query option not supported""
                         }
+                        ],
```

Comment:
Can you add a scenario that mimics legacy error responses (essentially that do not have the additive changes)
This is a runtime component that is being shared with all services for every API version released and this change is especially crucial when we talk about Multi-API packages.